daemons/gptp/linux/src/linux_hal_common.cpp

```diff
@@ -178,6 +178,7 @@ static void x_readEvent(int sockint, IEEE1588Port *pPort, int ifindex)
 	struct msghdr msg = { (void *) &snl, sizeof snl, &iov, 1, NULL, 0, 0 };
 	struct nlmsghdr *msgHdr;
 	struct ifinfomsg *ifi;
+	static unsigned last_ifi_flags = 0;
```

Comment:
Doing it this way will break if multiple interfaces are ever used. We don't currently support multi-port usage, but we also don't want to make life difficult if we decide to implement it.